tests/test_connection.py

```diff
@@ -121,7 +121,7 @@ async def wrapper(*args, **kwargs):
         return wrapper
 
 
-@attr.s(hash=False, cmp=False)
+@attr.s(hash=False, eq=False)
```

Comment:
Oh, attr dependency is missing from `requirements-dev.txt`.  Please add it, with minimum version corresponding to `eq` API change.